plugins/fuse/plugin_tests/server_fuse_test.py

```diff
@@ -0,0 +1,484 @@
+        }
+        self.adminFileName = 'user/admin/Private/Item 1/File 1A'
+        self.publicFileName = 'user/user/Public/Item 3/File 3'
+        self.privateFileName = 'user/user/Private/Item 4/File 4'
+
+    def tearDown(self):
+        from girder.plugins.fuse import server_fuse
+
+        super(ServerFuseTestCase, self).tearDown()
+        if self.extraMount:
+            server_fuse.unmountServerFuse(self.extraMount)
+        os.rmdir(self.extraMountPath)
+
+    def testMainMount(self):
+        """
+        Test the default mount point has access to all of the expected files.
+        """
+        mountpath = self.mainMountPath
+        self.assertEqual(sorted(os.listdir(mountpath)), sorted(['user', 'collection']))
+        # Check that all known paths exist and that arbitrary other paths don't
+        for fullpath in self.knownPaths:
+            self.assertTrue(os.path.exists(os.path.join(mountpath, fullpath)))
+            self.assertFalse(os.path.exists(os.path.join(mountpath, fullpath + '.other')))
+            size = os.path.getsize(os.path.join(mountpath, fullpath))
+            if self.knownPaths[fullpath]:
+                with open(os.path.join(mountpath, fullpath)) as file1:
+                    self.assertEqual(file1.read().strip(), self.knownPaths[fullpath])
+                self.assertGreater(size, 0)
+            stat = os.stat(os.path.join(mountpath, fullpath))
+            # The mtime should be recent
+            self.assertGreater(stat.st_mtime, time.time() - 1e5)
+            path = fullpath
+            # All parents should be folders and have zero size.
+            while '/' in path:
+                path = path.rsplit('/')[0]
+                self.assertTrue(os.path.isdir(os.path.join(mountpath, path)))
+                self.assertFalse(os.path.exists(os.path.join(mountpath, path + '.other')))
+                if '/' in path:
+                    self.assertEqual(os.path.getsize(os.path.join(mountpath, path)), 0)
+
+    def testUserMount(self):
+        """
+        The first non-admin user should be able to see all of their own items,
+        folders, collections, and files, but not the admin items or private
+        folders.
+        """
+        from girder.plugins.fuse import server_fuse
+
+        mountpath = self.extraMountPath
+        self.extraMount = 'test'
+        self.assertTrue(server_fuse.mountServerFuse(
+            self.extraMount, mountpath, level=AccessType.READ, user=self.user))
+        # The OS can cache stat, so wait 1 second before using the mount.
+        time.sleep(1)
+        self.assertEqual(sorted(os.listdir(mountpath)), sorted(['user', 'collection']))
+        for fullpath in self.knownPaths:
+            canSee = not fullpath.startswith('user/admin')
+            self.assertEqual(os.path.exists(os.path.join(mountpath, fullpath)), canSee)
+            if canSee and self.knownPaths[fullpath]:
```

Comment:
can we pull this into separate tests so we're testing the user can see what they're supposed to see,  and can't see what they're not supposed to see?